processor/src/main/java/org/mapstruct/ap/spi/DefaultAccessorNamingStrategy.java

```diff
@@ -16,7 +16,7 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-package org.mapstruct.ap.internal.naming;
+package org.mapstruct.ap.spi;
```

Comment:
I reckon its good enough. Besides, as long as we don't make functional changes.. Remember that its also good to have examples. Although I think that most developers are smart enough to look one package further :smile:.